### PR TITLE
Add Submission 125 Poster

### PR DESCRIPTION
**Pull request by mdhaber**
Submission 125 Poster

@mckib2 can you suggest changes to `info.json` to add your affiliation and ORCID?

### CODE REVIEW

**Comment by scoobies** [comment]
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

File: presentations/posters/125_haberland/info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "authors": [
+        {
+            "name": "Matt Haberland",
+            "affiliation": "California Polytechnic State University, San Luis Obispo",
+            "orcid": "0000-0003-4806-3601"
+        },
+        {
+            "name": "Nicholas McKibben",
+            "affiliation": "?",
+            "orcid": "0000-0000-0000-000?"
```
Comment (mckib2) [inline presentations/posters/125_haberland/info.json]:
```suggestion
            "affiliation": "Skayl, LLC",
            "orcid": "0000-0003-0034-2230"
```

**Comment by mdhaber** [comment]
@scoobies help

**Comment by deniederhut** [comment]
> hundreds of submissions, and dozens of volunteers

🙃

**Review by deniederhut — APPROVED** [review]

**Pull request merged**